### PR TITLE
Find suitable camera devices

### PR DESCRIPTION
**Pull request by amutake**
## Check list

- [x] I wrote [CHANGELOG.md](./CHANGELOG.md) if the pull request adds/modifies features.

## Summary

We added `find_usb_camera_device` and `find_csi_camera_device` in order to find suitable camera device from Actcast apps.

### Motivation

In bullseye, when both CSI camera and USB camera are connected, the device files numbered the following patterns.

- CSI camera: `/dev/video0` (Video Capture) and `/dev/video1` (Video Metadata), USB camera: `/dev/video2` (Video Capture) and `/dev/video3` (Video Metadata)
- USB camera: `/dev/video0` and `/dev/video1`, CSI camera: `/dev/video2` and `/dev/video3`
- CSI camera: `/dev/video0`, USB camera: `/dev/video1` and `/dev/video2`
- USB camera: `/dev/video0` and `/dev/video1`, CSI camera: `/dev/video2`

To make it easier to select the desired camera from Actcast apps, we have implemented the two functions.

### CODE REVIEW

File: actfw_core/v4l2/video.py [hunk 1]
```diff
@@ -870,10 +870,10 @@ def _ioctl(self, request, arg):
     def query_capability(self):
         cap = capability()
         result = self._ioctl(_VIDIOC.QUERYCAP, byref(cap))
-        if not (cap.capabilities & _V4L2_CAP_VIDEO_CAPTURE):
-            raise RuntimeError("The device doesn't support the single-planar API through the Video Capture interface.")
-        if not (cap.capabilities & _V4L2_CAP_STREAMING):
-            raise RuntimeError("The device doesn't support the streaming I/O method.")
+        if not (cap.device_caps & _V4L2_CAP_VIDEO_CAPTURE):
```
Comment (amutake) [inline actfw_core/v4l2/video.py:873]: In bullseye, multiple device files can be created for a single physical device. Only one of these device nodes can be used for video capture, so we have changed `capabilities` (physical device capabilities) to `device_caps` (device node capabilities).
This has been tested and confirmed to work on both buster and bullseye.

ref: https://www.kernel.org/doc/html/v5.15/userspace-api/media/v4l/vidioc-querycap.html#c.V4L.v4l2_capability

File: actfw_core/system.py [hunk 2]
```diff
@@ -218,3 +220,41 @@ def get_camera_device_info(default_image_source: Optional[str] = None) -> Device
...
+        for node in c.nodes:
+            paths.append(str(node.path))
+    return paths
+
+
+def _find_specific_video_device(video_port: VideoPort) -> Optional[str]:
+    devs = _list_video_devices()
+    for dev in devs:
+        try:
+            with Video(dev) as video:
```
Comment (riku179) [inline actfw_core/system.py:239]: video.close() されないとエラーになることを書いた方が良さそう

**Review by riku179 — APPROVED** [review]
LGTM with nits 👍

File: actfw_core/system.py (hunk 2 above)
Comment (amutake) [reply actfw_core/system.py:240]: 書きました :pray:

**Review by riku179 — APPROVED** [review]

**Pull request merged**